NEWS

```diff
@@ -48,6 +48,12 @@
 		- Removing redundant and fixing implicit imports [Thoralf Klein]
 		- Hide many methods from SWIG, reducing compile memory by 500MiB [Heiko Strathmann, Fernando Iglesias, Thoralf Klein]
 
+2014-12-16 Fernando Iglesias <fernando.iglesias@shogun-toolbox.org>
```

Comment:
Wait I am confused. Why did you add this here? Looking at the file doesnt seem it has the correct structure now